### DISCUSSION TITLE
03-08-24 - Reunion

### DISCUSSION THREAD

**Discussion by alejovillores** [trabajo-profesional-fiuba/assignment-service#148] [General]
# Temas a debatir

Algunos temas para debatir y llegar a diferentes conclusiones

## Backend

- [x] Conexion con front
- [x] Conexion con modulos de computo
- [x] Grupos
- [x] Cuatrimestres
- [x] Arquitectura (opiniones y mejoras)

**Comment by alejovillores** [comment]
**Front**
Como administrador\
Quiero poder crear un nuevo cuatrimestre

Como administrador\
Quiero poder ver los cuatrimestres como cards\
Para subir los csvs

Como administrador\
Quiero poder ver los datos cargados por cuatrimestre

Redux para estados

Ver Google Analytics.

**Backend**

GET tutors
GET students (todos todos en caso de lista vacia)
CLEAR tablas cada vez que se sube un csv si ya hay registros
POST tema 
Agregar capacidad en relacion tema-tutor: (many-to-many)  --> TEMA,TUTOR,CAPACIDAD entonces en la relacion del many to many va

- Preguntar a Carlos tema de capacidades.

**Comment by celedituro** [comment]
- Agregar cuatrimestres en endpoint
- Actualizar archivos: eliminar y agregar cuando un archivo se sube por 2da vez
- Agregar capacidad de tutor al cargar archivo de tutores
- Agregar tutor y capacidad (si el tema es único o no) en creación de temas
